source/directives/rhForm.js

```diff
@@ -43,6 +44,17 @@ module.exports = [ "$compile", function ( $compile )
                         scope.save( scope.model );
                     }
                 };
+                scope.getShowDeleteState = function ()
+                {
+                    // NOTE: Maintain Default for Previous Implementation
+                    if ( !(typeof( scope.showDelete ) === 'boolean' || angular.isFunction( scope.showDelete )) )
+                    {
+                        return true;
+                    }
```

Comment:
I'm not sure even with the comment that I understand what this block is for. If it makes sense to have a breaking change, that's fine, we'll just bump the major version for release